Formula/geotoad.rb

```diff
@@ -0,0 +1,21 @@
+class Geotoad < Formula
+  desc "Query tool to query the geocaching.com website"
+  homepage "https://github.com/steve8x8/geotoad"
+  url "https://github.com/steve8x8/geotoad/archive/3.25.4.tar.gz"
+  sha256 "6c30bb7ce6ae552b4146ac1c228c18c9e3cf318860ff0fc709cee803406ce900"
+  head "https://github.com/steve8x8/geotoad.git"
+
+  def install
+    mv "geotoad.rb", "geotoad"
+    libexec.install "data", "geotoad", "interface", "lib", "templates"
+
+    bin.write_exec_script "#{libexec}/geotoad"
+
+    info.install "FAQ.txt", "Templates.txt"
```

Comment:
These are non-essential but are worth to keep around. Is `info.install` the correct one?